include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -553,26 +573,35 @@ class argument_parser
             return;
         }
 
+        bool special_format_was_set{false};
+        bool sub_parser_was_set{false};
```

Comment:
Why is this needed? Can't you check if  `unique_ptr != nullptr` ?

Reply:
left over from previous solutions :see_no_evil: